linux/network_device_ops/pvrdma_client_network_device_ops.yml

```diff
@@ -38,21 +38,16 @@
       vars:
         vm_power_state_set: 'restarted'
 
-    - name: "Get client VM's guest IP"
-      block:
-        - name: "Refresh client VM's guest IP till it has different IP with server VM"
-          include_tasks: refresh_pvrdma_client_vm_ip.yml
-          with_list: "{{ range(1, 11) | list }}"
-          loop_control:
-            loop_var: retry_count
-      when: >
-        (not pvrdma_client_vm_guest_ip) or
-        (pvrdma_client_vm_guest_ip == pvrdma_server_vm_guest_ip)
+    - name: "Refresh client VM's guest IP till it has different IP with server VM"
+      include_tasks: refresh_pvrdma_client_vm_ip.yml
+      with_list: "{{ range(1, 11) | list }}"
+      loop_control:
+        loop_var: retry_count
```

Comment:
Outside when doesn't break the loop even though the condition meets. It will always check 10 times. So I moved it inside so that it can break the loop when client guest IP is not same as server IP

Reply:
Searched and I saw writing in this way can break the loop.
```
    - name: "test task"
      include_tasks:
        file: test_include_task.yml
        apply:
           when: test_var < 3
      with_list: "{{ range(1, 6) | list }}"
      loop_control:
         loop_var: loop_count
```

Reply:
Thanks. Will try it.

Reply:
It worked. Thanks!